libraries/freertos_plus/standard/tls/src/iot_tls.c

```diff
@@ -518,6 +522,7 @@ static int prvInitializeClientCredential( TLSContext_t * pxCtx )
 
     if( xCertObj == CK_INVALID_HANDLE )
```

Comment:
Even though ```CK_INVALID_HANDLE``` is defined to ```0```, for readability it will be better to initialize ```xCertObj``` to ```CK_INVALID_HANDLE``` instead of ```0```.

```
CK_OBJECT_HANDLE xCertObj = CK_INVALID_HANDLE;
```

The same thing about ```pxCtx->xP11PrivateKey```. It will be better to add the following line to ```TLS_Init```:

```
pxCtx->xP11PrivateKey = CK_INVALID_HANDLE;
```

Reply:
Fixing the first one with the xCertObject, and changing (in conjunction w/ comment above) so that we are only looking at what the value of pxCtx->xP11PrivateKey is if we have explicitly set it.